content/docs/1.6.0/deploy/install/install-with-fleet.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Install with Fleet
+weight: 10
+---
+
+## Prerequisites
+- Your workstation: Install [Helm](https://helm.sh/docs/) v2.0 or later.
+- Kubernetes cluster:
+  - Ensure that each node fulfills the [installation requirements](../#installation-requirements).
+  - If you are using a Helm version earlier than v3.0, [install Tiller with role-based access control (RBAC)](https://v2.helm.sh/docs/using_helm/#tiller-namespaces-and-rbac).
```

Comment:
No need to have a note for helm v2 if we just recommend v3.

Reply:
Removed.